back/src/forms/resolvers/FormCompany.ts

```diff
@@ -2,7 +2,7 @@ import { FormCompanyResolvers } from "../../generated/graphql/types";
 
 const formCompanyResolvers: FormCompanyResolvers = {
   country: parent => {
-    return parent.country || "FR";
+    return parent.siret ? parent.country || "FR" : null;
```

Comment:
```suggestion
    return parent.vatNumber ? parent.country : parent.siret ? "FR" : null;
```
permettrait éventuellement de moins dépendre du fait que le `vatNumber` est copié dans le champ `siret`

Reply:
Ou éventuellement utiliser les helpers `isSiret` et `isVat`